pkg/cache/cache.go

```diff
@@ -31,6 +31,8 @@ const (
 	InstanceTypesAndZonesTTL = 5 * time.Minute
 	// InstanceProfileTTL is the time before we refresh checking instance profile existence at IAM
 	InstanceProfileTTL = 15 * time.Minute
+	// ExpandedSubnetTTL is time to allow reconciliation to update the subnet data needed before expiration
```

Comment:
I feel like this name is a little strange. Why not just create variables that match the cache name? e.g. `availableIPAddressTTL` and `associatePublicIPAddressTTL`